keras_cv/layers/fpn/feature_pyramid.py

```diff
@@ -0,0 +1,156 @@
+# Copyright 2022 The KerasCV Authors
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     https://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+import tensorflow as tf
+
+
+@tf.keras.utils.register_keras_serializable(package="keras_cv")
+class FeaturePyramid(tf.keras.layers.Layer):
+    """Implements a Feature Pyramid Network.
+
+    Feature Pyramid Networks (FPNs) are basic components that are added to an
+    existing CNN to extract features at different scales. For the basic FPN, the
+    inputs are features `Ci` from different levels of a CNN, where the feature is
+    scaled from the image size by `1/2^i` for any level `i`.
+
+    There is an output associated with each level in the basic FPN. The output Pi
+    at level `i` (corresponding to Ci) is given by performing a merge operation on
+    the outputs of:
+    1) a top down operation on Pj (where j = i - 1)
+    2) a lateral operation on Ci
+
+    The layer must be called with the feature Ci in increasing order of `i`. As an
+    example, for `pyramid_levels`=[2,3,4,5], the layer should be called with the
+    list [C2,C3,C4,C5] as input
+
+    Args:
+        pyramid_levels: a python list that specifies all the values of level `i`
+            for which the feature Ci will be specified. The size of this list is
+            num_pyramid_levels.
```

Comment:
let's quote num_pyramid_levels with `` so that its rendered as a param. Same for all the occurrences below.